File: src/main/java/com/example/appjamback/domain/user/domain/repository/UserRepository.java
package com.example.appjamback.domain.user.domain.repository;

import com.example.appjamback.domain.user.domain.User;
import org.springframework.data.jpa.repository.JpaRepository;

public interface UserRepository extends JpaRepository<User, Long> {
    User findByAccountId(String accountId);
}

File: src/main/kotlin/com/example/instargram/domain/user/domain/repository/UserRepository.kt
package com.example.instargram.domain.user.domain.repository

import com.example.instargram.domain.user.domain.User
import org.springframework.data.jpa.repository.JpaRepository
import java.util.UUID

interface UserRepository : JpaRepository<User, Long> {
    fun findByAccountId(accountId: String): User?
    fun existsByAccountId(accountId: String): Boolean

    fun findByRe(accountId: String): User?
}
